gips/data/landsat/landsat.py

```diff
@@ -1346,3 +1393,200 @@ def _readraw(self, asset_type):
 
         verbose_out('%s: read in %s' % (image.Basename(), datetime.now() - start), 2)
         return image
+
+    def sentinel2_coreg_export(self, tmpdir, utm_zone):
+        landsat_shp = landsatRepository.get_setting('tiles')
+        sentinel2Data = utils.import_data_class('sentinel2')
+        sentinel2Asset = sentinel2Data.Asset
+        spatial_extent = SpatialExtent.factory(sentinel2Data, site=landsat_shp, where="pr = '%s'" % self.id, pcov=33.0)[0]
+        temporal_extent = TemporalExtent(self.date.strftime("%Y-%j"))
+
+        self._time_report("querying for most recent sentinel2 images")
+        inventory = DataInventory(sentinel2Data, spatial_extent, temporal_extent, fetch=False)
+
+        # If there is no available sentinel2 scene on that day, search before and after
+        # until one is found.
+        delta = timedelta(1)
+        date_found = self.date
+        while len(inventory) == 0:
```

Comment:
While I think the likelihood of nothing ever being found is somewhat low, you are correct that this can be an infinite loop. I think the simplest method would to either set max retries or to set a maximum temporal distance from the starting date. @ircwaves any thoughts?

Reply:
Double coverage from GIPS-Dev check-in:

1. add upper bound on the size of the search window -- extreme case of +/- 90 days.
2. for Landsat scenes prior to S2 launch, use `2017-${SCENE_DOY}` as the center point for the search window.